src/controllers/auth.controller.js

```diff
@@ -0,0 +1,66 @@
+const authService = require('../services/auth.service');
+
+const register = async (req, res) => {
+  const userBody = req.body;
+  const newUser = await authService.register(userBody);
+
+  res.status(201).send(newUser);
+};
+
+const activate = async (req, res) => {
+  const { activationToken } = req.params;
+  const user = await authService.activate(activationToken);
```

Comment:
Consider adding error handling for the 'activate' function. If 'authService.activate' fails, it could result in an unhandled promise rejection.